### ISSUE TITLE
[Breadcrumb] Missing aria-label="Breadcrumb" on nav element

### ISSUE THREAD

**Issue by jakublamprecht** [adobe/aem-core-wcm-components#581]
## Bug Report

**Current Behavior**
Breadcrumb component misses aria-label attribute with value Breadcrumb on \<nav> tag as specified by W3C in the following links:

https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html
https://www.w3.org/TR/wai-aria-practices/#breadcrumb

**Expected behavior/code**
Aria label is present on the \<nav> tag.

**Environment**
- Core Components version 2.3.0

**Possible Solution**
```
<nav class="cmp-breadcrumb"
     data-sly-use.breadcrumb="com.adobe.cq.wcm.core.components.models.Breadcrumb"
     data-sly-use.template="core/wcm/components/commons/v1/templates.html"
     data-sly-test="${breadcrumb.items.size > 0}"
     aria-label="breadcrumb">
```

**Comment by richardhand** [comment]
Thanks @jakublamprecht for the suggestion! If you like, feel free to go ahead and make a pull request with this change.

It is also recommended to highlight current, which we could also do for the list item representing the current page.

**Comment by richardhand** [comment]
Code is merged to [master](https://github.com/adobe/aem-core-wcm-components/commits/master) and will be available in the upcoming [2.6.0 Release](https://github.com/adobe/aem-core-wcm-components/milestone/16).
